package.json

```
@@ -57,6 +58,7 @@
     "@openmrs/esm-framework": "next",
     "@openmrs/esm-patient-common-lib": "next",
     "@openmrs/esm-styleguide": "next",
+    "@playwright/test": "^1.36.1",
```

```suggestion
    "@playwright/test": "1.34.0",
```
We need to specify the exact version number that we use in the playwright container in bamboo job.

---

oh okay